File: abc/abc293/a.cpp
// https://atcoder.jp/contests/abc293/submissions/39601891

#include <bits/stdc++.h>
using namespace std;

int main() {
  string s;
  cin >> s;

  for(int i = 0; i <= s.size() / 2; i++) {
    swap(s[2 * i - 2], s[2 * i - 1]);
  }

  cout << s;
}

File: abc/abc293/a.java
// https://atcoder.jp/contests/abc293/submissions/43238706

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    String s = stdIn.next();
    for(int i = 0; i < s.length(); i += 2) {
      System.out.printf("%c%c", s.charAt(i + 1), s.charAt(i));
    }
  }
}
